client/components/mma/shared/benefits/BenefitsConfiguration.ts

```diff
@@ -22,6 +22,11 @@ const adFree = {
 	description: 'Avoid ads on all your devices',
 };
 
+const guardianWeekly = {
+	name: 'Guardian Weekly',
+	description: 'Print magazine delivered to your door every week ',
```

Comment:
was the space interntional?
```suggestion
	description: 'Print magazine delivered to your door every week',
```